app/package.json

```diff
@@ -54,7 +54,6 @@
     "react-dom": "^16.8.4",
     "react-transition-group": "^4.4.1",
     "react-virtualized": "^9.20.0",
-    "registry-js": "^1.15.0",
```

Comment:
obviously problematic as local runners are not getting this.. but just toying with ideas